packages/common/src/components/FileTabs/index.tsx

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
-import { Container, Tab, Close } from './elements';
+import { Tabs, Tab, Close } from './elements';
+
+/**
+  Because we have two themes, we need to
+  segregate application of styles.
+
+  We wrap the Container with a ThemeProvider.
+
+  TODO: Find a better way of doing this, we're
```

Comment:
We can add an extra `ThemeProvider` in the root (where we have `styled-components`' one), no? 🤔